### ISSUE TITLE
Please update to latest version of Protobuf

### ISSUE THREAD

**Issue by mradamfrancis** [adform/adform-tracking-ios-sdk#14]
I'm currently using Firebase SDK, which uses Protobuf. Firebase uses a minimum version of 3.1 of Protobuf, but adform-tracking requires version 3.0.0-alpha-4.1.
```
[!] Unable to satisfy the following requirements:

- `Protobuf (= 3.5.0)` required by `Podfile.lock`
- `Protobuf (~> 3.1)` required by `FirebaseRemoteConfig (2.1.0)`
- `Protobuf (~> 3.1)` required by `FirebaseABTesting (1.0.0)`
- `Protobuf (= 3.0.0-alpha-4.1)` required by `AdformTracking (1.1)`
```

**Comment by vladasdrejeris** [comment]
Hello,

We are currently working to solve this issue. We will let you know as soon as we have a new SDK version with a fix.

**Comment by vladasdrejeris** [comment]
This issue is solved in the new version of the SDK [1.1.5](https://github.com/adform/adform-tracking-ios-sdk/releases/tag/v.1.1.5)
